packages/db-ra-data-rest/src/index.spec.js

```diff
@@ -1,9 +1,10 @@
+import { vi } from 'vitest';
```

Comment:
oh my god, vitest has an awesome plain and simple configuration compared to jest